src/test/java/com/microsoft/graph/content/BatchRequestContentTest.java

```diff
@@ -21,13 +22,14 @@
 import java.nio.charset.StandardCharsets;
 import java.util.*;
 
+import static com.microsoft.graph.CoreConstants.ReplacementConstants.USERS_ENDPOINT_WITH_REPLACE_TOKEN;
 import static org.junit.jupiter.api.Assertions.*;
 import static org.mockito.Mockito.*;
 
 class BatchRequestContentTest {
-
-    static final String requestUrl = "https://graph.microsoft.com/v1.0/me";
+    static final String requestUrl = "https://graph.microsoft.com/v1.0"+USERS_ENDPOINT_WITH_REPLACE_TOKEN;
```

Comment:
Just to be sure and catch possible regressions maybe we can add a test with the endpoint as `/beta`

Reply:
Testing for beta endpoint can be seen in the Kiota-Java pr which now contains the UrlReplaceHandler.